cmd/producer/main.go

```diff
@@ -50,92 +47,74 @@ type myRedis struct {
 	client redis.Cmdable
 }
 
-// request size limit in bytes
+// Request size limit in bytes.
 const bitsInMB = 1000000
```

Comment:
the name of the const is a bit misleading and not correct (number of bits in MB would start with 8 most likely :))
Also you can move the const up with the other consts.

Reply:
🤦‍♀️ thanks - bytes as the comment says.